models/resnet.py

```diff
@@ -164,11 +164,21 @@ class ResNet(Model):
         Block (BasicBlock|BottleneckBlock): block module of model.
         depth (int): layers of resnet, default: 50.
         num_classes (int): output dim of last fc layer, default: 1000.
+        with_pool (bool): use pool or not. Default: False.
+        classifier_activation (str): activation for the last fc layer. Default: 'softmax'.
     """
 
-    def __init__(self, Block, depth=50, num_classes=1000):
+    def __init__(self,
+                 Block,
+                 depth=50,
+                 num_classes=-1,
+                 with_pool=False,
```

Comment:
Why default value is -1 and False?  Whether we keep the defalut value for ImageNet?

Reply:
done, thanks.